test/specs/test scenarios.js

```diff
@@ -0,0 +1,144 @@
+
+const faker = require("@faker-js/faker");
+const expectChai = require("chai").expect;
+
+describe('On Awesome-shop', () => {
+    it('user could add a product to the cart', async () => {
+      await browser.url(`https://awesome-shop.ru/`);
+
+      const item = await $('=Apple Cinema 30"');
+      await item.click();
+      const textForItem = await item.getText();
+        
+      await $("label*=Medium").click();
+      await $("label*=Checkbox 2").click();
+      await $("label*=Checkbox 4").click();
+        
+      await $("#input-option208").setValue("some new text");
+      const valueForTextInput = await $("#input-option208").getText();
+
+      await $("select#input-option217").click();
+      await $("option*=Green").click();
+        
+      await $("#input-option209").setValue("some new text some new text");
+      const valueForTextArea = await $("#input-option209").getValue();
+
+      let testValueForTextArea = "";
+      if (valueForTextArea.length > 20) {
+            testValueForTextArea = await valueForTextArea.slice(20)
+      }
+      else {testValueForTextArea = await valueForTextArea; }
```

Comment:
1) Please use following pattern for if-else:
```
if (condition) {
  // some actions for condition
} else {
  // some actions for non-condition
}
```
2) Line 28: forgotten semicolon.